docs/vs-2015/snippets/cpp/VS_Snippets_Misc/NVC_MFC_DrawClient/cpp/drawvw.cpp

```diff
@@ -135,9 +135,9 @@ BOOL CDrawView::PreCreateWindow(CREATESTRUCT& cs)
 	return TRUE;
 }
 
-void CDrawView::OnActivateView(BOOL bActivate, CView* pActiveView, CView* pDeactiveView)
+void CDrawView::OnActivateView(BOOL bActivate, CView* pActiveView, CView* pDeactivateView)
```

Comment:
I don't think these changes are correct. I think "deactive view" is as opposed to "active view" in the previous argument of the method.

Reply:
OK, maybe it should be "Inactive" since "Deactive" isn't a real word